backend/de.metas.business/src/main/java-legacy/org/compiere/model/CalloutOrder.java

```diff
@@ -296,6 +296,17 @@ public String docType(final ICalloutField calloutField)
 		return NO_ERROR;
 	}
 
+	public String shipper(final ICalloutField calloutField)
```

Comment:
instead of adding it here, please add the method to `de.metas.order.callout.C_Order` and have I_C_Order as parameter